spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/src/main/resources/META-INF/native-image/serialization-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Delete files that have no actual content.

Reply:
Done